sql/src/main/java/com/vmware/ddlog/util/sql/CalciteUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.vmware.ddlog.util.sql;
+
+import org.apache.calcite.avatica.util.Casing;
+import org.apache.calcite.sql.parser.SqlAbstractParserImpl;
+import org.apache.calcite.sql.parser.ddl.SqlDdlParserImpl;
+
+import java.io.StringReader;
+
+public class CalciteUtils {
+
+    /**
+     * Initialize and return a Calcite DDL Parser.
+     * @param sql  Statement to initialize parser with.
+     */
+    public static SqlAbstractParserImpl createCalciteParser(String sql) {
+        SqlAbstractParserImpl ret = SqlDdlParserImpl.FACTORY.getParser(new StringReader(sql));
+
+        /*
+         * Currently, SqlDdlParserImpl doesn't have a constructor that accepts config parameters,
```

Comment:
These days set returns the object itself, so you can chain these calls. ret.setBlah().setBleh().